src/lib.rs

```diff
@@ -147,34 +147,39 @@ pub use crate::primitives::absolute_locktime::{AbsLockTime, AbsLockTimeError};
 pub use crate::primitives::relative_locktime::{RelLockTime, RelLockTimeError};
 pub use crate::primitives::threshold::{Threshold, ThresholdError};
 
-/// Public key trait which can be converted to Hash type
+/// Trait representing a key which can be converted to a hash type.
 pub trait MiniscriptKey: Clone + Eq + Ord + fmt::Debug + fmt::Display + hash::Hash {
-    /// Returns true if the pubkey is uncompressed. Defaults to `false`.
+    /// Returns true if the key is serialized uncompressed, defaults to `false`.
```

Comment:
In 12dd99d20223270524000ea9cd421fe7af7b132d:

This is a run-on sentence (and it was not before).